MapleServer2/Types/EffectTriggers.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MapleServer2.Types;
+
+public struct EffectTriggers
+{
+    public bool IsEvent = false;
+    public int SkillId = 0;
+    public int EffectId = 0;
+    public IFieldActor Owner = null;
+    public IFieldActor Target = null;
+    public IFieldActor Caster = null;
```

Comment:
you dont need to specify the values here either. By default they're still those given values that you're assigning them.

Reply:
I tried it, but got compiler errors complaining about it